sdk/spring/azure-spring-boot-core/src/main/java/com/azure/spring/core/AzureCloud.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.core;
+
+/**
+ * Enum of the Azure cloud to connect to.
+ */
+public enum AzureCloud {
```

Comment:
how about change to AzureCloudEnv or AzureCloudEnvironment？

Reply:
will remove this enum and make environment property as a String value for extension.